index.html

```diff
@@ -22,7 +22,7 @@
 </header>
 <div class="container">
       
-  <div class="containers">
+  <div class="container">
```

Comment:
The  semantic tag divs are meant to be inside the section tags. Please rearrange it.